types/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+file(GLOB TYPES_FILES
+    ${CMAKE_CURRENT_SOURCE_DIR}/*.json
+)
+
+add_custom_target(generate_type_files
+    DEPENDS
+        ${GENERATED_OUTPUT_DIR}/types
+)
+
+# FIXME (aw): right now, the header files are generated in-source for convenience
```

Comment:
Where does this comment comes from? I guess it's stale

Reply:
I found the exact same comment in interfaces/CMakeLists.txt which was the base for this file